src/swap/components/Swap.tsx

```diff
@@ -21,6 +21,7 @@ export function Swap({
   onError,
   onStatus,
   onSuccess,
+  isSponsored,
```

Comment:
could default to false here and not have to handle undefined

Reply:
I don't think we need special handling for undefined since it's falsey

Reply:
but I can add a default here

Reply:
added in latest diff

Reply:
I still like explicitly defining as false